package.json

```diff
@@ -97,7 +97,8 @@
   },
   "dependencies": {
     "classnames": "2.3.1",
-    "react-textarea-autosize": "8.3.2"
```

Comment:
Why did we remove `react-textarea-autosize`?

Reply:
It should be a mistake. Seems like our `Textarea` component is still using it. I added again.